contrib/download-frozen-image.sh

```diff
@@ -42,6 +42,8 @@ while [ $# -gt 0 ]; do
 	[ "$tag" != "$imageTag" ] || tag='latest'
 	tag="${tag%@*}"
 
+	imageFile="${image////_}" # "/" can't be in filenames :)
```

Comment:
That sounds sane, but I think we need to test that under Bash 3 in Windows before we say for sure: (this is my testing on Bash 4 in Linux)
```console
$ a='a/b/c'
$ echo "${a//\//_}"
a_b_c
```